runtime-sdk/modules/evm/src/precompile/mod.rs

```diff
@@ -99,14 +100,14 @@ impl<Cfg: Config, B: EVMBackendExt> PrecompileSet for Precompiles<'_, Cfg, B> {
     }
 
     fn is_precompile(&self, address: H160) -> bool {
-        // All Ethereum precompiles are zero except for the last byte, which is no more than five.
-        // Otherwise, when confidentiality is enabled, Oasis precompiles start with one and have a last byte of no more than four.
+        // All Ethereum precompiles are zero except for the last byte, which is no more than six.
```

Comment:
Is having this additional `is_precompile` fn really a worth optimization (vs just letting the above `match` handle this)?

Reply:
Yeah I think it would be better to have a common match function instead of this, if possible to do nicely.

Reply:
Unless I'm missing something, just removing the `is_precompile` function would already work like that, no?